setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
 ignore = H301
-exclude = venv*,logo,docs,build
+exclude = venv*,logo,docs,build,target,.pybuilder
+line-length = 120
```

Comment:
Can we postpone this change on coding style and discuss it in a PR about coding style guideline (e.g. optimized imports)?

Reply:
Removed it. BTW, flake8 default line length is 120. Python way: "better explicit then implicit" ;-)